test-backend.sh

```diff
@@ -13,6 +13,7 @@ set -e
 #
 
 export GOBIN=${PWD}/bin:${GOBIN}
+export GOFLAGS=-mod=vendor
```

Comment:
is this really needed here? perhaps another PR? it looks to me valid, but fully unrelated to what we are doing in this PR.

Reply:
another PR opened https://github.com/openshift/console/pull/4007